sycl/include/sycl/detail/helpers.hpp

```diff
@@ -10,8 +10,8 @@
 
 #include <CL/__spirv/spirv_types.hpp> // for MemorySemanticsMask
 #include <sycl/access/access.hpp>     // for fence_space
-#include <sycl/detail/export.hpp>     // for __SYCL_EXPORT
-#include <sycl/memory_enums.hpp>      // for memory_order
+#include <sycl/detail/export.hpp> // for __SYCL_EXPORT
```

Comment:
Unnecessary?

Reply:
Indeed, reverted those changes in cf21174495433bc5d0f061bea5cf1f926859d7b1